src/components/tooltip/tooltip.jsx

```diff
@@ -21,56 +16,65 @@ export default function Tooltip({
   text,
   truncate,
 }) {
-  const [visible, setVisible] = useState(false);
+  // size of the triangle in pixels; this should be the same as teh value in the linked stylesheet
+  // TODO is there a way to share the variable so its only in one place
+  const triangleHeight = 8;
```

Comment:
@juanca I know you told me there was a way to share variables between css and jsx but I couldn't figure out how to do it.

I thought it was just something like `<div styles={{ '--triangle-height': triangleHeight }} />` but when I did that I never saw the value actually used in my css, I might have done something wrong though.. thoughts?

Reply:
Huh.... I can lend my 👀  cuz it should work.

Reply:
Ok, figured it out - I just forgot the units 🤦 

I was using `<div styles={{ '--triangle-height': triangleHeight }} />` when what I should have had was `<div styles={{ '--triangle-height': `${triangleHeight}px` }} />`